numpy/f2py/tests/test_array_from_pyobj.py

```diff
@@ -549,6 +549,10 @@ def test_in_cache_from_2casttype_failure(self):
                 continue
             if t.elsize >= self.type.elsize:
                 continue
+            is_int = np.subdtype(t.dtype, np.integer)
+            if is_int and int(self.num2seq[0]) > np.iinfo(t.dtype).max:
+                # skip test if num2seq would trigger an overflow error
+                continue
```

Comment:
Before this PR one of the tests that uses this function silently relied on being able to create an int8 array from `['11111', '22222']`.

Reply:
OK, probably fine to skip uint8/int8, here.